### DISCUSSION TITLE
Make a spatialite jdbc connection

### DISCUSSION THREAD

**Discussion by geostonemarten** [dbeaver/dbeaver#10179] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version : Windows 10 64x
- DBeaver : Latest

#### Connection specification:
- Database name and version : Spatialite 3 (SQLite extension spatiale

#### Describe your question:
How can I configure en SQLITE with module spatialite ou directly a Spatialite JDBC connector

This is related to **#6830 #5960**

**Comment by kseniiaguzeeva** [comment, marked as answer]
This article should help you https://github.com/dbeaver/dbeaver/wiki/Database-drivers
